enhancements/deprecated-bundles.md

```diff
@@ -25,60 +25,49 @@ status: implementable
 
 ## Summary
 
-OPM will support marking versions as deprecated and truncating the update graph of a given package. Deprecated versions will not be installable. If a deprecated version is installed prior to upgrade appropriate alerts will be fired on cluster. 
+OPM will support marking versions as deprecated for a given package - no bundles will be removed from the index during deprecation. Deprecated versions will not be installable. (Optional) If a deprecated version is installed prior to upgrade appropriate alerts will be fired on cluster.
 
 ## Motivation
 
-The default set of Operators provided on OCP clusters will soon (4.6+) be delivered through indexes and each OCP minor version will have a unique set of index images. Today, packages in one index will be duplicated to the next minor version index when it is available. This means operator owners releasing through the redhat, certified, or community pipelines will now need a way to specify which versions of their operator are incompatible or unsupported in the next minor version index.
+The default set of Operators provided on OCP clusters will soon (4.6+) be delivered through indexes and each OCP minor version will have a unique set of index images. Today, packages in one index will be duplicated to the next minor version index when it is available unless the team know ahead of time that a bundle is not supported on a given OCP version. This means operator owners releasing through the redhat, certified, or community pipelines will need a way to specify which versions of their operator are incompatible or unsupported in the next minor version index. All deprecated bundles still need to be present in their respective indexes (even if uninstallable) in order for the bootstrap process of new OCP minor version specific index images from old index images to succeed.
 
 ### Terminology
 
 #### Deprecated bundle
 
 A deprecated bundle is an upgradable but not installable bundle in the update graph of an operator.
 
-#### Terminated bundle
-
-A terminated bundle is neither upgradable nor installable.
-
 ### Goals
 
-- A deprecated bundle in the update graph and any of its replacements (terminated bundles) will not be installed by OLM
+- A deprecated bundle in the update graph and any of its replacements will not be installed by OLM
 - Operator Owners don’t need to release a new version in order to specify which bundle is deprecated
 - Index maintainers have a way to un-do deprecation if needed
 - The deprecation is specified after promotion (not during) and before publishing of the index image
-- Alerts are fired on cluster when a deprecated version is installed
+- (Optional) Alerts are fired on cluster when a deprecated version is installed
```

Comment:
Can you elaborate here? How is it optional, and how is that controlled?

Reply:
will add a story about how the deprecated property results in a CSV status update on cluster.

Reply:
given the urgency of fixing how deprecation works(so it does not remove bundles), i'd be inclined to leave this out of this iteration entirely so we don't burn too many cycles on discussing it or debating implementing it.